blazor/dropdown-list/value-binding.md

```diff
@@ -78,9 +80,9 @@ The following code demonstrates array of int as datasource to the DropDownList c
 
 ## Object binding
 
-object data binding refers a memory location where object data is stored.
+You can bind Object data to the [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute of the DropdownList component, i.e You can map the class name to `TValue`. 

```

Comment:
Bind the Object data to the [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute of the DropdownList component, this is, you can map the class name to the `TValue.`